server.js

```diff
@@ -0,0 +1,21 @@
+require("dotenv").config();
+
+const { createApp } = require("./app");
+const  AppDataSource  = require("./src/models/data-source");
+
+const setServer = async() => {
+  const app = createApp();
+  const PORT = process.env.PORT;
+
+    await AppDataSource.initialize();
```

Comment:
```suggestion
    await appDataSource.initialize();
```
.then().catch()를 활용하여 실제 콘솔에서 db pooling이 잘 되었는지 확인 할 수 있게 해주세요 :)